src/ModelBinding/BindExtensions.cs

```diff
@@ -62,6 +62,17 @@ public static T Bind<T>(this HttpRequest request)
                         }
                         return val.Value.Select(y => Convert.ChangeType(y, colType));
                     }
+
+                    // Convert.ChangeType (below) chokes on Guids, so doing it here instead.
+                    if (propertyType == typeof(Guid))
```

Comment:
will this work if it is `Guid?`

Reply:
Good question! Let me come back to you.

Reply:
Ok, I have pushed the changes now. Are you able to see them?

Reply:
Ok, I can see that the changes are here. Let me know if you find anything else wrong with it. I did a test with both `Guid` and `Guid?` and it worked fine now.